CONTRIBUTING.md

```diff
@@ -6,14 +6,49 @@ Contributions are welcome!
 
 Before contributing to this repository for the first time, please review our project's [Code of Conduct](https://github.com/devfile/api/blob/main/CODE_OF_CONDUCT.md).
 
-## Getting Started
+## Certificate of Origin
+
+By contributing to this project you agree to the Developer Certificate of
+Origin (DCO). This document was created by the Linux Kernel community and is a
+simple statement that you, as a contributor, have the legal right to make the
+contribution. See the [DCO](DCO) file for details.
+
+In order to show your agreement with the DCO you should include at the end of the commit message,
+the following line:
+
+```console
+Signed-off-by: Firstname Lastname <email@email.com>
+```
+
+Once you set your user.name and user.email in your git config, you can sign your commit automatically with `git commit -s`.
+
+## How to Contribute:
 
 ### Issues
 
 - Open or search for [issues](https://github.com/devfile/api/issues) with the label `area/api`.
 
 - If a related issue doesn't exist, you can open a new issue using a relevant [issue form](https://github.com/devfile/api/issues/new/choose). You can tag issues with `/area api`.
 
+### Submitting Pull Request
+
+When you think the code is ready for review, create a pull request and link the issue associated with it.
+
+Owners of the repository will watch out for new PRs and provide reviews to them.
+
+If comments have been given in a review, they have to be addressed before merging.
+
+After addressing review comments, don't forget to add a comment in the PR with the reviewer mentioned afterward, so they get notified by Github to provide a re-review.
```

Comment:
Might be good to mention that for the `devfile/api` reviews from `CODEOWNERS` are required?

Reply:
@thepetk @yangcao77 How is this change? https://github.com/devfile/api/pull/1597/commits/bd2157e65d7a156e489c940927d463b0efaba072